Update `getting-started.md` to reflect changes for SRI 1.5.0 (#279)
stratum-mining/stratumprotocol.org: app/src/_developers/getting-started.md

Reviewer: We need to adjust lines 81-83.

There's no `channel_nominal_hashrate` parameter anymore.

Also, we need to tell the user to add their username in the `user_identity` field of `jdc-config-hosted-example.toml` file. I would also add this suggestion in the paragraph related to optional adjustments.

Reviewer: We also need to adjust lines 19-22 accordingly

 
`./minerd -a sha256d -o stratum+tcp://localhost:34255 -q -D -P`

#### Translator Proxy and Job Declarator configurations (optional)

##### - Aggregated vs. Non-Aggregated mode

In Stratum V2, a **channel** represents a dedicated mining session between a downstream mining device and an upstream node (such as a pool, job declarator, or proxy). Each channel has its own mining target, accepts work submissions, and is uniquely identified by a `channel_id`.

You can configure whether the Translator Proxy (TProxy) manages channels in **aggregated** or **non-aggregated** mode using the `aggregate_channels` parameter in `tproxy-config-local-jdc-example.toml`.

* **Non-Aggregated (aggregate_channels: `false`)** – Each mining device connected to the Translator Proxy opens its own upstream channel. This provides isolation per device and let the Pool to have monitoring capabilities on individual mining devices.
* **Aggregated (aggregate_channels: `true`)** – All miners connected to the Translator Proxy share a single upstream channel. This mode reduces significantly the bandwidth usage, but doesn't let the Pool to have monitoring capabilities on individual mining devices.

> For a detailed explanation of channels, see the [Stratum V2 specification, Section 5.2](https://stratumprotocol.org/specification/05-Mining-Protocol/#52-channel).

Reviewer: No need for this since here we are doing JD

#### Adjust proxy-config (optional)

> Depending on mining device you do run, you may have to adjust `tproxy-config-local-jdc-example.toml` file in order to adjust the `min_individual_miner_hashrate` parameter accordingly.

Reviewer: We are missing the `user_identity` modification on JDC config file here.

Reviewer: I would add the section as you did later.

Author: Section added to the three setups, adapted accordingly

```bash
tar xvf bitcoin-sv2-tp-0.1.19-x86_64-linux-gnu.tar.gz
```

⚠️ Note: macOS binaries are not code signed. Read release notes for instructions on how to proceed.

Start the Template Provider.

```bash
./bitcoin-sv2-tp-0.1.19/bin/bitcoind -testnet4 -sv2 -sv2port=8442 -debug=sv2 
```

⚠️ Note: you need to wait until `bitcoind` is fully synced with the testnet4 before you proceed.

Optional parameters:

There are optional parameters which can be used to better manage the Template Provider:
- `sv2interval` - sets how often a new template is built (default is 30s)
- `sv2feedelta` - defines the delta fees to reach before sending new templates to downstreams (default is 1000 sats)
- `loglevel=sv2:trace` to get more detailed debugging

For example: 

```bash
./bitcoin-sv2-tp-0.1.19/bin/bitcoind -testnet4 -sv2 -sv2port=8442 -sv2interval=20 -sv2feedelta=1000 -debug=sv2 -loglevel=sv2:trace
```
This way new templates are constructed every 20 seconds (taking the most profitable txs from the mempool) and they are sent downstream if new fees collected are more than 1000 sats. 

#### Clone the repository

```bash

> git clone https://github.com/stratum-mining/stratum.git

Author: Isn't after the 1.6.0?

Because in the 1.5.0 we will still be in the older structure.

And the tutorial is trying to point to the latest stable release.

cd roles/pool/config-examples
cargo run -- -c pool-config-local-tp-example.toml
```

#### Run Translator Proxy

```bash
cd roles/translator/config-examples/
cargo run -- -c tproxy-config-local-pool-example.toml
```
#### Connect Mining Devices

Connect mining device - Translator Proxy will be running on port `34255`, so you will need just to edit your mining device(s) configuration, adding the line:

`stratum+tcp://<tProxy ip>:34255`
#### CPU Miner
If you don't have a physical miner, you can do tests with CPUMiner.

Setup the correct CPUMiner for your OS:
- You can download the binary directly from [here](https://sourceforge.net/projects/cpuminer/files/);
- Or compile it from [https://github.com/pooler/cpuminer](https://github.com/pooler/cpuminer)

On the CPUMiner directory:
 
`./minerd -a sha256d -o stratum+tcp://localhost:34255 -q -D -P`

#### Aggregated vs. Non-Aggregated Mode for TProxy (optional)

In Stratum V2, a **channel** represents a dedicated mining session between a downstream mining device and an upstream node (such as a pool, job declarator, or proxy). Each channel has its own mining target, accepts work submissions, and is uniquely identified by a `channel_id`.

> You can configure whether the Translator Proxy (TProxy) manages channels in **aggregated** or **non-aggregated** mode using the `aggregate_channels` parameter in `tproxy-config-local-jdc-example.toml`.

Reviewer: ```suggestion
You can configure whether the Translator Proxy (TProxy) manages channels in **aggregated** or **non-aggregated** mode using the `aggregate_channels` parameter in `tproxy-config-local-pool-example.toml`.
```

* **Non-Aggregated (default: `false`)** – Each mining device connected to the TProxy opens its own upstream channel. This provides isolation per device and is generally the preferred setup.

> * **Aggregated (`true`)** – All miners connected to the TProxy share a single upstream channel. This mode exists mainly for compatibility with Stratum V1-style proxies but can also reduce network overhead in certain setups.

Reviewer: ```suggestion
* **Aggregated (aggregate_channels: `true`)** – All miners connected to the Translator Proxy share a single upstream channel. This mode reduces significantly the bandwidth usage, but doesn't let the Pool to have monitoring capabilities on individual mining devices.
```

> 

Reviewer: Here we need the paragraph related to optional adjustments where we tell the user to set the `user_identity` and `min_individual_miner_hashrate` in the config files.

For a detailed explanation of channels, see the [Stratum V2 specification, Section 5.2](https://stratumprotocol.org/specification/05-Mining-Protocol/#52-channel).

> ## III Getting Started - Running all roles 

Reviewer: ```suggestion
## III Getting Started - Running all roles with Job Declaration (JD)
```

 
`./minerd -a sha256d -o stratum+tcp://localhost:34255 -q -D -P`

#### Translator Proxy and Job Declarator configurations (optional)

##### - Aggregated vs. Non-Aggregated mode

In Stratum V2, a **channel** represents a dedicated mining session between a downstream mining device and an upstream node (such as a pool, job declarator, or proxy). Each channel has its own mining target, accepts work submissions, and is uniquely identified by a `channel_id`.

You can configure whether the Translator Proxy (TProxy) manages channels in **aggregated** or **non-aggregated** mode using the `aggregate_channels` parameter in `tproxy-config-local-jdc-example.toml`.

* **Non-Aggregated (aggregate_channels: `false`)** – Each mining device connected to the Translator Proxy opens its own upstream channel. This provides isolation per device and let the Pool to have monitoring capabilities on individual mining devices.
* **Aggregated (aggregate_channels: `true`)** – All miners connected to the Translator Proxy share a single upstream channel. This mode reduces significantly the bandwidth usage, but doesn't let the Pool to have monitoring capabilities on individual mining devices.

> For a detailed explanation of channels, see the [Stratum V2 specification, Section 5.2](https://stratumprotocol.org/specification/05-Mining-Protocol/#52-channel).

Reviewer: This is not needed in the JD scenario, remove it

##### - User Identity

> You may also want to adjust the `user_identity` parameter in `tproxy-config-local-jdc-example.toml` and `jdc-config-local-example.toml`. This parameter defines an identifier that will be visible to the upstream apps. A counter is automatically appended for each connected client (e.g., `username.miner1`, `username.miner2`).

Reviewer: ```suggestion
You may also want to adjust the `user_identity` parameter in `tproxy-config-local-jdc-example.toml` and `jdc-config-local-example.toml`.
```
